plugins/ipc-rules/ipc-rules-common.hpp

```diff
@@ -195,8 +195,22 @@ static inline wf::json_t view_to_json(wayfire_view view)
     description["minimized"]   = toplevel ? toplevel->minimized : false;
     description["activated"]   = toplevel ? toplevel->activated : false;
     description["sticky"]     = toplevel ? toplevel->sticky : false;
-    description["wset-index"] = toplevel && toplevel->get_wset() ? toplevel->get_wset()->get_index() : -1;
```

Comment:
It would be enough to cast the result of get_index to `int64_t` instead of writing a long lambda xD